zstash/extract.py

```diff
@@ -350,7 +350,7 @@ def multiprocess_extract(
     workers_to_matches: List[List[FilesRow]] = [[] for _ in range(num_workers)]
     for db_row in matches:
         tar = db_row.tar
-        workers_idx: int
+        worker_idx: int  # type: ignore
```

Comment:
This may not be the ideal way to solve this error, but I don't really want to change the code when I'm not familiar with the overall context.